CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- Fixed some unsoundness around thread-safety on Windows. 
```

Comment:
There should be a newline between the header and the list, and this entry is a bit vague.